.github/push-image.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 1 ]; then
+    echo "Pushes a container image to Docker Hub with tags"
+    echo 
+    echo "Usage: $0 PROJECT"
+    echo "Example: $0 dummy-api"
+    echo "Relies on the following environment variables:"
+    echo "- GITHUB_REF, GITHUB_SHA (GH Action default)"
+    echo "- DOCKER_USERNAME, DOCKER_PASSWORD"
+    echo "Uses PROJECT:latest as input image."
+    echo "Override this behaviour with IMAGE_TAG environment variable."
+    exit 1
+fi
+
+PROJECT="$1"
+IMAGE_TAG="${IMAGE_TAG:-latest}"
```

Comment:
I see the point in making as many things as possible optional, but not aware of a use case for this default as builds that favor reproducibility always build a commit sha tag. Only locally you would have a default but it would still be `local` to prevent accidental clashes with the `latest` that CI has pushed e.g. someone wanting to use the official one and having that overridden by a local image built from a random branch.

Reply:
My thinking was that this would be useful for simpler pipelines e.g.:

```bash
docker build . -t libero/product
./scripts/github/push-image.sh product
```
But I see you point. If I remove the `latest` as default, wouldn't it be nice to pass the `IMAGE_TAG` explicitly? As in `push-image.sh project tag` or even `push-image.sh project:tag`.

Reply:
Yes, can be made explicit. Two separate arguments are easier to deal with.